### PR TITLE
Add support for named components (fixes #48)

### LINKED ISSUE #48: Accomodate multiple components in a route?
In the sidebar example, multiple components are specified for a route.
https://github.com/rackt/react-router/blob/master/examples/sidebar/app.js#L102

Should this be possible with relay? Would the `queries` parameter need to accept an object mapping queries to the multiple components? Like:

```jsx
<Route components={{sidebar: Sidebar, content: Content}}
       queries={{sidebar: sidebarQueries, content: contentQueries}}>...</Route>
```

 Or, perhaps the queries could be available for all the (relay) components.
```jsx
<Route components={{sidebar: Sidebar, content: Content}}
       queries={routeQueries}>...</Route>
```

### PR DESCRIPTION
**Pull request by denvned**
Fixes https://github.com/relay-tools/react-router-relay/issues/48.
Docs: https://github.com/reactjs/react-router/blob/v2.0.1/docs/API.md#named-components

### CODE REVIEW

File: src/RouteAggregator.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import Relay from 'react-relay';
 
 import getParamsForRoute from './getParamsForRoute';
 
+const DEFAULT_KEY = 'default';
```
Comment (taion) [inline src/RouteAggregator.js]: Make this `@@default` just in case.

File: src/RouteAggregator.js [hunk 2]
```diff
@@ -35,40 +37,55 @@ export default class RouteAggregator {
 
       const component = components[i];
```
Comment (taion) [inline src/RouteAggregator.js]: Maybe rename this to `routeComponent` just to avoid the `no-shadow` stuff below?

File: test/RelayRouter.spec.js [hunk 3]
```diff
@@ -19,15 +19,12 @@ describe('<RelayRouter>', () => {
...
+        const { widget, first, second } = this.props;
 
         return (
-          <div>
-            <div className={route.className}>
-              {widget.name}
-            </div>
-
-            {children}
+          <div className={widget.name}>
```
Comment (taion) [inline test/RelayRouter.spec.js]: :+1:

**Comment by taion** [comment]
Two nits, then this is good to go.

Thanks!

**Comment by denvned** [comment]
Done!

**Comment by taion** [comment]
My dependencies are messed up. I'll fix them and re-run CI.

**Comment by taion** [comment]
I'll resolve merge conflicts by hand.

**Comment by taion** [comment]
Merged in https://github.com/relay-tools/react-router-relay/commit/f6e6e72402a16ea87113cf3f38ea5b86a700810b.

**Pull request closed without merging**